src/sweeper/address_parser.py

```diff
@@ -148,6 +162,15 @@ def normalize_street_type(type_text):
     raise InvalidStreetTypeError(type_text)
 
 
+HWY_REGEX = compile('(SR|STATE ROUTE|HIGHWAY)')
+def normalize_street_name_pre_type(text):
+    '''normalizes highways by doing things like replaces SR with HWY and removes US
+
+    No need to worried about casing or "."s because usaddress has already taken care of them by this point.
+    '''
+    return HWY_REGEX.sub('HWY', text).replace('US ', '')
```

Comment:
do you need `flags=re.IGNORECASE` here?

will this replace `woodsrow` with `woodhighwayow`?

Reply:
It should be ALL CAPS at this point in the code, but yes, I probably need to add word boundaries or something...

Reply:
I added a test for this, but didn't need to add any source code. Since this function is working on the street name prefix type, the street name never gets run through it.